.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -58,6 +58,34 @@ jobs:
           commit_message: 🤖 Bump ${{ github.event.inputs.release-type }} version of Airbyte CDK
           commit_user_name: Octavia Squidington III
           commit_user_email: octavia-squidington-iii@users.noreply.github.com
+      - name: Post failure to Slack channel dev-connectors-extensibility
+        if: !success()
+        uses: slackapi/slack-github-action@v1.23.0
+        continue-on-error: true
+        with:
+          channel-id: C02TL38U5L7
+          payload: |
+            {
+                "text": "Error while publishing Airbyte CDK!"
```

Comment:
Any reason not to give a more specific message depending on whether the error occurred during `bump-version`, `build-cdk`, or `publish-cdk`?

Reply:
We can. I was checking the process and I think that in any case, the user will have to click on the link of the workflow to have more detail and therefore, he will have the information anyway. If we think it has value to be added earlier (like in the Slack message), let me know and I can add it

Reply:
I think that extra bit of detail would be nice to have.